mungegithub/submit-queue/www/index.html

```diff
@@ -34,7 +34,7 @@ <h2 class="md-toolbar-tools">E2E Tests Failing. Entire submit queue blocked. Las
           <md-tab label="PRs" md-on-select="cntl.selectTab('prs')">
             <md-content class="md-whiteframe-z2">
               <md-toolbar>
-                <h2 class="md-toolbar-tools">Pull Requests</h2>
+                <h2 class="md-toolbar-tools">Pull Requests ({{cntl.prsCount}})</h2>
```

Comment:
I'm pretty stupid here, but does `{{cntl.prs.length}}` work without the .js change?  Also, do we care that the 'search' will change the pr list but not this count?